delta.tera

```diff
@@ -0,0 +1,42 @@
+---
+whiskers:
+  version: 2.4.0
+  filename: "catppuccin.gitconfig"
+  hex_format: "#{{r}}{{g}}{{b}}"
+---
+
+{%- for identifier, flavor in flavors -%}
+{%- set palette = flavor.colors -%}
+
+[delta "catppuccin-{{ identifier }}"]
+	blame-palette = "{{ palette.base.hex }} {{ palette.mantle.hex }} {{ palette.crust.hex }} {{ palette.surface0.hex }} {{ palette.surface1.hex }}"
+	commit-decoration-style = "box ul"
+	{{ if(cond=flavor.dark, t="dark", f="light") }} = true
+	file-decoration-style = "{{ palette.text.hex }}"
+	file-style = "{{ palette.text.hex }}"
+	hunk-header-decoration-style = "box ul"
+	hunk-header-file-style = "bold"
+	hunk-header-line-number-style = "bold {{ palette.subtext0.hex }}"
+	hunk-header-style = "file line-number syntax"
+	line-numbers = true
+	line-numbers-left-style = "{{ palette.overlay0.hex }}"
+	line-numbers-minus-style = "bold {{ palette.red.hex }}"
+	line-numbers-plus-style = "bold {{ palette.green.hex }}"
+	line-numbers-right-style = "{{ palette.overlay0.hex }}"
+	line-numbers-zero-style = "{{ palette.overlay0.hex }}"
+	# 25% red 75% base
+	minus-emph-style = "bold syntax {{ palette.red | mix(color=palette.base, amount=0.25) | get(key="hex") }}"
+	# 10% red 90% base
+	minus-style = "syntax {{ palette.red | mix(color=palette.base, amount=0.1) | get(key="hex") }}"
+	# 25% green 75% base
+	plus-emph-style = "bold syntax {{ palette.green | mix(color=palette.base, amount=0.25) | get(key="hex") }}"
+	# 10% green 90% base
+	plus-style = "syntax {{ palette.green | mix(color=palette.base, amount=0.1) | get(key="hex") }}"
+	map-styles = \
+		bold purple => syntax "{{ palette.mauve | mix(color=palette.base, amount=0.25) | get(key="hex") }}", \
+		bold blue => syntax "{{ palette.blue | mix(color=palette.base, amount=0.25) | get(key="hex") }}", \
+		bold cyan => syntax "{{ palette.sky | mix(color=palette.base, amount=0.25) | get(key="hex") }}", \
+		bold yellow => syntax "{{ palette.yellow | mix(color=palette.base, amount=0.25) | get(key="hex") }}"
+	# Should match the name of the bat theme
+	syntax-theme = "Catppuccin {{ identifier | capitalize }}"
+{% endfor -%}
```

Comment:
nit: Will a newline here make the sections separated in `catppuccin.gitconfig`? Would be nice.

Reply:
Yes it would. For these Whiskers PRs we generally try to avoid changing stuff like this to make the diff easier to read, so the maintainer of this repository could change that later down the road if they wish.

Reply:
Opened https://github.com/catppuccin/delta/pull/11 for the newline change

Reply:
Emphasis on "down the road"! Opening PRs *before* this is merged is kinda silly ahah.